openid-data-classes/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/IdentifierListInfo.kt

```diff
@@ -0,0 +1,59 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList
+
+import at.asitplus.wallet.lib.data.rfc3986.UniformResourceIdentifier
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.cbor.ByteString
+
+/**
+ * DER encoded X509 Certificate
+ */
+private typealias Certificate = ByteArray
```

Comment:
Is it sensible to use a private type alias in a public data class? I suggest dropping that type alias.